rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -5207,11 +5207,11 @@ void Compiler::CreateBranch(llvm::Value * cmp_i1, llvm::Value * target_i32, bool
 			}
 
 			SetPc(target_i32);
-//			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
+			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
```

Comment:
I think you should leave it commented so that it doesn't confuse people ; It's better not to have non executed code for readability.